clustering.ipynb

```diff
@@ -64,6 +64,8 @@
     "import pandas as pd\n",
```

Comment:
<p>What is the resulted intuition from this plot? </p><p>There may be another plot to be used for these categorical variables that result better!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Reema-Khaseeb/DM-Assignment1-NYC-transportation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
bar plots are the best way to visualize categorical data based on research.

Reply:
<p>This is not a barplot</p>